prow/perf_service_graph_test.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+#!/bin/bash
+WD=$(dirname "$0")
+WD=$(cd "$WD"; pwd)
+ROOT=$(dirname "$WD")
+
+# Exit immediately for non zero status
+set -e
+# Check unset variables
+set -u
+# Print commands
+set -x
+CMD=""
+DELETE=""
+source "${ROOT}/prow/lib.sh"
+
+TIME_TO_RUN_PERF_TESTS=${TIME_TO_RUN_PERF_TESTS:-1200}
+
+pushd "${GOPATH}/src/istio.io/tools/perf/load"
+  WD="${GOPATH}/src/istio.io/tools/perf/load"
+  source common.sh
+  start_servicegraphs "1" "0"
+popd
+# Run the test for some time
+echo "Run the test for ${TIME_TO_RUN_PERF_TESTS}"
+
+kubectl -n istio-system port-forward $(kubectl get pod --namespace istio-system --selector="app=prometheus" --output jsonpath='{.items[0].metadata.name}') 8060:9090 > /tmp/forward &
+
+sleep 5s
+
+OUTPUT_PATH=${OUTPUT_PATH:-"/tmp/output"}
+
+mkdir -p "${OUTPUT_PATH}"
+
+perf_metrics="${OUTPUT_PATH}/perf_metrics.txt"
+rm "${perf_metrics}" || true
+
+pushd "${GOPATH}/src/istio.io/tools/perf/benchmark/runner"
+  count=$(expr $TIME_TO_RUN_PERF_TESTS / 60)
+  echo "Get metric $count time(s)."
+  for i in `seq 1 $count`;
+  do
+    sleep 1m
+    python prom.py http://localhost:8060 60 --no-aggregate >> ${perf_metrics}
```

Comment:
So our end result will be 10 json objects?
```
{
... 1st minute
}
{
... 2nd minute
}
```

?

Just verifying.

Reading/analyzing these will be done in a later PR or do we have something for that now?

Reply:
Correct. 
We are not going to use the data here, just save it in a bucket together with all the artifacts for that particular daily build. 
I will set up daily cron job for reading the data in another script.